package-lock.json

```diff
@@ -1203,9 +1203,8 @@
             "dev": true
         },
         "@exabyte-io/esse.js": {
-            "version": "2023.6.20-0",
-            "resolved": "https://registry.npmjs.org/@exabyte-io/esse.js/-/esse.js-2023.6.20-0.tgz",
-            "integrity": "sha512-KFab2+Z8BDKXWDuK7DYFYEpyaOLUespLIxZwqaJ4fKJQRk7KZ6U0FmXGkZVFdrmPgAzwMOhDO2M04pYB7q4OYw==",
+            "version": "git+https://github.com/Exabyte-io/esse.git#943ee219ad712777092819156e6be63f533112f3",
```

Comment:
Was there a change in ESSE in this task? If not - why is the package-lock updated??

Reply:
No change in ESSE compared to the epic branch. This must have been introduced when the epic was merged into this branch.

Reply:
I don't understand the answer - if no changes, this diff should not be present in the PR

Reply:
I meant that during the merge I must have resolved the conflict by "Accept Theirs". Anyways, the two files are in fact identical as `git diff --name-status epic/SOF-6009` shows:
```
M       src/utils/filter.js
M       src/utils/object.js
M       tests/utils/filter.tests.js
M       tests/utils/object.tests.js
```
I.e. `package-lock.json` is not marked as modified. This is a GitHub artifact.